### ISSUE TITLE
Enable `Raw Stack Trace` as default

### ISSUE THREAD

**Issue by axilis-marko** [getsentry/sentry#66946]
### Problem Statement

I prefer always seeing Raw Stack Trace. So when I'm jumping thorugh Issues, I need to go to `...` and under Display enable `Raw stack trace` for every of those Issues. I would like to always see Raw Stack Trace.

### Solution Brainstorm

This as a global user setting seems to be the easiest solution (together with "Most Relevant" and "Full Stack Trace").

Propagating this setting through one user session also seems like an okay solution. So setting it on one Issue would apply to other Issues while going through them.

### Product Area

Issues

**Comment by leeandher** [comment]
Thanks for writing in @axilis-marko! I'll pass this along to the team to prioritize. 

In the meantime though, do you have any feedback for how we can improve the 'Most Relevant' stack trace view?

**Comment by axilis-marko** [comment]
Not really, it's a great feature. It's just that with Raw Stack Trace I'll have the same view I get in other logs, so it's easier to switch between Loki for example and Sentry.

**Comment by revolutio-au** [comment]
I wholeheartedly support this, the stacktrace in raw format is my primary use Sentry and is why I moved across from my own error reporting module. It would be great to avoid 2 unnecessary clicks to get the data I need every time I need to investigate an issue. A simple option in the Settings, or having Sentry remember and use the view settings from the last time it was changed, would be extremely useful (and I imagine quite simple to implement).
